apps/src/aichat/views/modelCustomization/PublishNotes.tsx

```diff
@@ -126,6 +128,33 @@ const PublishNotes: React.FunctionComponent = () => {
           className={modelCustomizationStyles.updateButton}
         />
       </div>
+      {havePropertiesChanged && !saveInProgress && !isReadOnly && (
+        <Alert
+          text={'You have unsaved changes'}
+          type={'warning'}
+          size="s"
+          className={modelCustomizationStyles.saveAlert}
+        />
+      )}
+      {emptyModelCard && !saveInProgress && !isReadOnly && (
+        <Alert
+          text={'Remember to save your changes'}
+          type={'info'}
+          size="s"
+          className={modelCustomizationStyles.saveAlert}
+        />
```

Comment:
This logic does have an unlikely case where if you previously had changes, and then clear all of the fields, prior to clicking "Save" you'll see BOTH the "Remember to save your changes" and "You have unsaved changes" notifications. I have a pending [question](https://codedotorg.slack.com/archives/C06FELVTV0Q/p1724268489106159?thread_ts=1723562419.689669&cid=C06FELVTV0Q) to Design, but would be happy to get reviewers opinions, too! I was trying to avoid having to track "Save" clicks and get too complicated. 

One alternative is to just have two states "saved" and "unsaved" where saved is inclusive of empty/you haven't done anything yet.

Reply:
talked it through with Design... updating so that if both the saved changes _and_ the current changes are empty, then we show the "Remember to save changes" notification, which results in only ever showing one at a time